packages/frontend/src/components/TableCalculationModels/TableCalculationModal.tsx

```diff
@@ -120,14 +121,19 @@ const TableCalculationModal: FC<Props> = ({
                 onSubmit={(data: TableCalculationFormInputs) => {
                     const { name, sql } = data;
                     try {
-                        onSave({
-                            name: getUniqueTableCalculationName(
-                                name,
-                                tableCalculations,
-                            ),
-                            displayName: name,
-                            sql,
-                        });
+                        // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+                        sql?.includes('/') &&
+                        (targetDatabase === 'postgres' ||
+                            targetDatabase === 'redshift')
+                            ? setIsDividing(true)
+                            : onSave({
```

Comment:
will the onSave ever happen if input has `/` in the content?

Reply:
nope, should it ?

Reply:
@IrakliJani is right, I can't save the TC if / is present 

[Screencast from 23-12-22 16:32:49.webm](https://user-images.githubusercontent.com/1983672/209360853-2bff01c8-a221-47ce-9ac0-8f82b9a7ffda.webm)